### PR TITLE
 Add Eden-SDN example for Local NI with multiple ports 

### PR DESCRIPTION
**Pull request by milan-zededa**
This example demonstrates Local network instance with multiple ports and multipath static IP routes.
EVE API changes were introduced and described here:
https://github.com/lf-edge/eve-api/pull/53

### CODE REVIEW

**Comment by milan-zededa** [comment]
@uncleDecart PR is ready for review and merge (EVE 13.0 with support for multi-port Local NI was released).

File: sdn/examples/local-ni-multiple-ports/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+192.168.30.0/24 dev eth2 proto static scope link src 192.168.30.20 metric 1010
+```
+
+Notice that `eth0` is selected for both the default route and the HTTP server route.
+
+Login to the application and try to access something in the Internet (e.g. 8.8.8.8)
+and the HTTP server:
+
+```shell
+eve$ eve attach-app-console 3599588a-17d3-4d02-aae1-bcefe3706cfd.1.1/cons
```
Comment (uncleDecart) [inline sdn/examples/local-ni-multiple-ports/README.md:128]: this uuid `3599588a-17d3-4d02-aae1-bcefe3706cfd.1.1/cons` is hardcoded, right?

**Review by uncleDecart — APPROVED** [review]
LGTM

File: sdn/examples/local-ni-multiple-ports/README.md (hunk 1 above)
Comment (milan-zededa) [reply sdn/examples/local-ni-multiple-ports/README.md:128]: yes, hardcoded in the device config of the example

**Comment by milan-zededa** [comment]
> LGTM. Do you want to add any tests for this?

I want to automatize all the examples but there are some challenges...

**Pull request merged**